releases/point-2022-3.md

```diff
@@ -0,0 +1,81 @@
+* **Optimized speed and performance** when loading the Lifecycle Management details screen.
+* **Improved speed** for loading the Overview tile when accessing the Private Channel details screen.
+* **Improved messaging** when a Bad Request or Internal server error occurs that provides more insight into the issue that occurred when more information is available.
+* **Improved Access Review feature** to include exact dates when accessing the PR History Details Overview.
+  * Names of columns changed to include Status, Performed On and Performed By.
+    * Status bar states whether the action was Completed or Expired, and hovering above it provides information on who completed the task.
+* **Improved speed** for accessing the Lifecycle Management Email Preview.
+* **Fixed a bug** where the New Update Available ribbon was showing even after updating SysKit Point to the newest version.
+* **Fixed a bug** where the loading screen was cut off generating the details report.
+    * Once loading was completed, the report would be unresponsive when trying to select lines.
+* **Fixed an issue** where an error message was shown after opening the Details screen on Alerts.
+* **Fixed an issue** where removing members from a Group or Team through Microsoft 365 portal doesn’t sync properly.
+    * When all members from a group or team are removed, and the group or team is empty after syncing, SysKit Point didn’t update that action and left all the members that were there from the previous sync.
+* **Fixed a bug** where workdays wouldn’t show when scheduling an Access Review using 2k resolution.
+    * When trying to schedule an Access Review and entering the number of workdays allowed to complete the request, the number wouldn’t show in the selection box when using 2k resolution (2560x1440).
+* **Fixed an issue** where a large quantity of deleted groups caused problems with the automatic sync function.
+    * The automatic sync would process groups, and if a tenant has more than 1000 deleted groups, upon seeing they are deleted, the sync action would be canceled, and the state reverted to idle.  
+* **Fixed a bug** where the Users overview showed additional users that no longer existed in the Azure Active Directory.
+    * Users overview would show Orphaned users that were not detected as permanently deleted.
+* **Fixed an issue** where Sites Overview showed deleted sites as Active.
+* **Fixed an issue** with OneDrive delta sync, showing that almost all items are unique.
+    * After sync, all subitems of a folder would show as having unique permission, even though that was not the actual state of them on SysKit Point.
+* **Fixed a bug** where exporting a report for All Sites as an XLSX file failed due to detecting invalid characters.
+* **Fixed an issue** where using symbols, for example, &!@#, when creating a new team would show an error and crash the provisioning.
```

Comment:
prijedlog
* **Fixed an issue** where the provisioning of Microsoft Teams and Microsoft 365 Groups would fail if specific characters - for example, &,!,@, or # - were used as a part of the Team/ Group Name when requesting a new workspace in the SysKit Point Teams app.